# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/GeckoMVMContext.cpp
# Commit: 0e259884f052
# Full Hash: 0e259884f052230b24601efc3cde3a5ef5d4e6ad
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2019-05-01 15:54:31
# Regressor Bug: 1535945
# File Overlap Count: 1
# Description:
#   Bug 1535945 - Don't skip invalidating frames when creating displayports for async scrollable ancestors. r=tnikkel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D23816
# ==============================================================================

diff -r 732976d3f555 -r 0e259884f052 layout/base/GeckoMVMContext.cpp
--- a/layout/base/GeckoMVMContext.cpp	Wed May 01 08:16:44 2019 +0000
+++ b/layout/base/GeckoMVMContext.cpp	Wed May 01 04:23:07 2019 +0000
@@ -156,7 +156,7 @@
                                               displayportBase);
     nsIScrollableFrame* scrollable = do_QueryFrame(root);
     nsLayoutUtils::CalculateAndSetDisplayPortMargins(
-        scrollable, nsLayoutUtils::RepaintMode::DoNotRepaint);
+        scrollable, nsLayoutUtils::RepaintMode::Repaint);
   }
 }
 